libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/dateFilter/tests/__snapshots__/changeDateFilterSelectionHandler.test.ts.snap

```diff
@@ -16,6 +16,12 @@ Array [
 exports[`changeDateFilterSelectionHandler should set the date filter in the state when a date filter is changed 1`] = `
 Object {
   "dateFilter": Object {
+    "dateFilter": Object {
```

Comment:
This is extremely suspicious!

Reply:
Sorry I missed that in the review, should we revert the pull-request?.